schemas/alpha/azure.yaml.json

```diff
@@ -1331,6 +1309,158 @@
                     }
                 }
             }
+        },
+        "sqlDbResource": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type": true,
+                "uses": true,
```

Comment:
What does `"type": true` and `"uses": true` mean?

Reply:
Similar as above definitions, I think they indicate that the condition is always valid here (skip validation), and these two properties should follow the validation rules defined earlier.